skin/frontend/base/default/js/sd_recentlyviewed/recent.js

```diff
@@ -79,12 +79,17 @@ SomethingDigitalRecentlyViewed.prototype = {
 			return;
 		}
 
-		var html = rvTemplate.innerHTML.replace('{{items}}',this.getRenderedItems());
-		$(target).insert({top: html});
+		var insertParams = {}, //to enable use of variable keys (i.e. $position: content)
+			insertPosition = sdRecentlyViewed.insertPosition,
+			html = rvTemplate.innerHTML.replace('{{items}}',this.getRenderedItems());
+
+		insertParams[insertPosition] = html;
+
+		$(target).insert(insertParams);
```

Comment:
Agreed. I am not well-versed in the ways of Prototype, so this was just some jQuery habits shining through. ;) The intention was what you said.